src/handler.ts

```diff
@@ -0,0 +1,59 @@
+import { Handler, APIGatewayProxyEvent, Context, APIGatewayProxyResult } from 'aws-lambda';
+import * as req from 'request-promise';
```

Comment:
Don't use this or bluebird. These are ES5 libraries before Promises were native to JS